int8 SUCCESS=0
int8 EXCEPTION=1
int8 UNKNOWN_METHOD=2

rviz_msgs/UUID request_id
int8 error_code
string error_string

uint8[] data
